manifests/config.pp

```diff
@@ -9,6 +9,11 @@
           unless  => "test /etc/alternatives/java -ef '${java::use_java_alternative_path}'",
         }
       }
+      if $java::use_java_home != undef {
+        file { '/etc/environment':
```

Comment:
Yeah I wasn't really sure which way to go here - as it is the $java_home variable is being set, but unless I'm mistaken I don't think it's actually ever used. Is there no way for two different classes/modules to touch the same file without creating a resource conflict? (Like using line_in_file in ansible or something)